.github/workflows/master.yml

```diff
@@ -187,14 +191,25 @@ jobs:
           check-latest: false
       - name: Setup Maven
         uses: ./.github/actions/setup-maven
+      - name: Cache Engine Archives
+        uses: ./.github/actions/cache-engine-archives
       - name: Build on Scala ${{ matrix.scala }}
         run: |
-          MODULES='!externals/kyuubi-flink-sql-engine'
```

Comment:
This should also be retained to ensure the successful compilation of all Kyuubi modules, except for flink-engine, in the Scala-2.13 environment.